tests/testthat/test-variance_threshold_select.R

```diff
@@ -0,0 +1,44 @@
+#
+# Tests for variance_threshold_select()
+#
+library(tibble)
+
+testthat::test_that("relevant features remain", {
+  data <- tibble(
+    a = c(1,2,3),
+    b = c(2,2,2),
+    c = c('123', '123', '123'),
+    d = c('123', '124', '1')
+  )
+
+  feature_indexes <- variance_threshold_select(data)
+  testthat::expect_identical(feature_indexes, c(1, 4))
+})
+
+testthat::test_that("higher threshold", {
```

Comment:
Perhaps a bit more descriptive here.

```suggestion
testthat::test_that("appropriate feature is dropped with higher threshold", {
```